v3.2/glfw/util.go

```diff
@@ -21,3 +23,23 @@ func bytes(origin []byte) (pointer *uint8, free func()) {
 	ptr := C.CBytes(origin)
 	return (*uint8)(ptr), func() { C.free(ptr) }
 }
+
+func imageToGLFW(img image.Image) (r C.GLFWimage, free func()) {
+	b := img.Bounds()
+
+	r.width = C.int(b.Dx())
+	r.height = C.int(b.Dy())
+
+	if m, ok := img.(*image.NRGBA); ok && m.Stride == b.Dx()*4 && b.Min.X == 0 {
```

Comment:
Do we really need `b.Min.X == 0`?

Reply:
I think it does, but I'm not completely confident.

Reply:
Yeah, it can be done without that check as well.